src/activation_grid.py

```diff
@@ -14,113 +14,97 @@
 
 '''This module contains code to render the activation grid'''
 
-# do I keep this part? since we are just importing the functions in main, unsure if this needs to be here. commenting out for now
```

Comment:
@kjschmidt913 since this module is supposed to contain the methods for other code to call, I'd think no? I created model as an argument you pass into this class, so that way you could use ActivationGrid with any sort of model

Reply:
lmk if you like this design or if you think we should change it